examples/basf2/basf2_chain_example.py

```diff
@@ -56,10 +55,12 @@ def create_path(self):
         path = basf2.create_path()
 
         path.add_module('RootInput', inputFileNames=self.get_input_file_names("simulation_full_output.root"))
-        modularAnalysis.loadGearbox(path)
```

Comment:
One remark: I am 100% sure that this example was not working, since `add_reconstruction` needs the `Geometry` module for running and `modularAnalysis.loadGearbox` only appends the `Gearbox` module.

(Moreover, we plan to remove `modularAnalysis.loadGearbox`, since there are no reasons for using such module at analysis level.)